samples/Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Models/DummyResponseModel.cs

```diff
@@ -54,5 +54,7 @@ public class DummyResponseModel
         public DummySubResponseModel SubResponse2 { get; set; }
 
         public DummyGenericModel<DummyModel> DummyGenericDummyModel { get; set; }
+
+        public List<DummyResponseModel> Children { get; set; }
```

Comment:
Can we add a new `public Dictionary<string, DummyResponseModel> Children2 { get; set; }` property?